### DISCUSSION TITLE
My main issue with Auth is the need to manually add a Trigger to create the profiles table

### DISCUSSION THREAD

**Discussion by ftzi** [supabase/supabase#30843] [Feature Requests]
This is a big issue for me: https://supabase.com/docs/guides/auth/managing-user-data

Every app will basically need a profiles table and it's not really dev-friendly to do it.

Better Auth has an awesome way to handle custom data for users: https://www.better-auth.com/docs/concepts/database#extending-core-schema. After adding your data, you can run a command to update the Drizzle schema file and then you can run Drizzle to push the changes. Also, it's typesafe.

Can you please improve the profile creation/update to not use this manual SQL steps? I really want to use Supabase but this is really blocking me to have a general solution for my apps.

**Comment by saltcod** [comment]
This largely comes down to not being too prescriptive & opinionated. 
Many users use the trigger approach as described (see the public.handle_new_user() function in that doc you linked) 

This is pretty light-weight and reliable.
